testnets/prysmtestnet/chain.json

```diff
@@ -0,0 +1,158 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "prysmtestnet",
```

Comment:
If this is a devnet as opposed to a testnet, then the chain_name and directory name should also be 'prismdevnet'.
(sorry for the confusing instruction, I wasn't sure which it was)